FAQs.md

```diff
@@ -0,0 +1,121 @@
+## FREQUENTLY ASKED QUESTIONS
+
+This is the list of Frequently asked questions for Jockey music player. It is intended to serve as a self-service utility document. This is to aid new users or already existing users of Jockey music player.
+This document may be updated from time to time to suit any new changes to the software as new developments or changes will be implemented.
+
+1. **What is Jockey Music Player?**
```

Comment:
What are your thoughts on just calling the app "Jockey" and excluding the "music player" suffix?

Reply:
Well I know Jockey is the official app name. But adding the music player will give new users a hint to what this app is without having to read through anything.